lib/ansible/plugins/inventory/linode.py

```diff
@@ -158,8 +177,8 @@ def _validate_option(self, name, desired_type, option_value):
 
         return option_value
 
-    def _get_query_options(self, config_data):
-        """Get user specified query options from the configuration."""
+    def _get_user_options(self, config_data):
```

Comment:
I'm not sure why you changed this? It could be described both ways, I suppose. 

Hence, this feels like it could be bikeshed :sweat_smile:.

Reply:
"query" felt like this was only related to configuring the API query, which it was with region/type. Adding `use_public_ip` isn't related to query, just config.

Reply:
Got it, agreed!